<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QDialog" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="userList">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>11</y>
     <width>281</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="challangeButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Challange</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QLabel" name="challangedLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
